.github/workflows/build-and-publish.yml

```diff
@@ -111,4 +111,10 @@ jobs:
         uses: rickstaa/action-create-tag@v1.2.0
         with:
           tag: ${{ steps.GetVersion.outputs.tag }}
+      - name: Queue Git Release
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: Git Release
+          token: ${{ secrets.PERSONAL_TOKEN }}
```

Comment:
we'll need to [pass in the ref](https://github.com/benc-uk/workflow-dispatch) as well `ref: ${{ steps.GetVersion.outputs.tag }}` (might need the ref/tags/ prefix, I'm not sure)